### PR TITLE
Document s3 config changes

### PR DESCRIPTION
**Pull request by ilya-hontarau**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [ ] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [ ] The preview looks fine.
- [ ] The tests pass.
- [ ] The commit history is clean and meaningful.
- [ ] The pull request is opened against the `main` branch.
- [ ] The pull request is no longer marked as a draft.
- [ ] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/product/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ description: Find out about the latest changes to the Self-Hosted Spacelift.
 
 # Changelog
 
+## Changes between v1.1.0-hotfix.1 and v1.2.0
+
+- [Added S3 config](install.md#s3-config)
```
Comment (TheMacies) [inline docs/product/administration/changelog.md:9]: If we could rephrase this a bit to be more specific - for example: "Extended Self Hosted installation config to support specifying retention periods of various s3 buckets"

**Review by TheMacies — APPROVED** [review]

File: docs/product/administration/install.md [hunk 2]
```diff
@@ -218,6 +218,22 @@ your desired tags to the `global_resources_tags` array in the _config.json_:
 ]
 ```
 
+##### S3 Config
+
+You can configure the following options for the S3 buckets, they are all required, but have prefilled values in the config.
+
```
Comment (peterdeme) [inline docs/product/administration/install.md:224]: Let's add a table here where we explain what these buckets do.

Comment (peterdeme) [reply docs/product/administration/install.md:224]: @ilya-hontarau I can do it too if you don' t have time. I'm doing a self hosted release currently so we need this

Comment (ilya-hontarau) [reply docs/product/administration/install.md:224]: @peterdeme I can, let me switch

Comment (ilya-hontarau) [reply docs/product/administration/install.md:224]: but will be good to know what do these buckets actually do

Comment (peterdeme) [reply docs/product/administration/install.md:224]: Run logs: this is where we store the logs of a run
Delivery: contains webhook and audit trail deliveries.
Large queue messages: SQS has a limitation of message size, we use an S3 bucket to work around it.
Metadata: contains metadata for run initialization.
Policy input: we store policy inputs here - this is used for Policy Sampling.
Uploads: used for uploading stack states during stack creation.
User uploaded workspace: used for storing code for the local preview feature.
Workspaces: the workspaces are stored here for paused runs (eg.: waiting for confirmation).
Access logs: access logs for the load balancer.

**Review by peterdeme — APPROVED** [review]

**Pull request merged**